runtime/interpreter/variable.go

```diff
@@ -20,4 +20,12 @@ package interpreter
 
 type Variable struct {
 	Value Value
+	Depth int
+}
+
+func NewVariable(value Value, depth int) *Variable {
+	return &Variable{
+		Value: value,
+		Depth: depth,
```

Comment:
Minor: Not sure if we would maybe want to call this `ActivationDepth` or `StackDepth` or something a bit more specific.

Reply:
Good idea! I'll rename to `ActivationDepth` 👍

Reply:
Refactored in d0bf87d, including for `sema.Variable`